### ISSUE TITLE
Not working when doing bundle with java

### ISSUE THREAD

**Issue by eeroan** [JulienCabanes/requirejs-tpl#5]
Everything works fine when using node for bundling, with command: 
node build/r-2.0.2.js -o build/booking.build.js

But when doing same with java and rhino like this:
java -classpath build/js.jar:build/compiler.jar org.mozilla.javascript.tools.shell.Main build/r-2.0.2.js -o build/booking.build.js

I get following error:

Tracing dependencies for: booking
TypeError: Cannot call property get in object [object Object]. It is not a function, it is "undefined".
In module tree:
    booking
      initDom
        tpl

TypeError: Cannot call property get in object [object Object]. It is not a function, it is "undefined".
In module tree:
    booking
      initDom
        tpl

I'm using latest version of closure compiler and rhino.
Version of require.js is 2.0.4 and r.js 2.0.2

**Comment by andywhite37** [comment]
Just wanted to confirm that I ran into the same issue with the tpl plugin when running r.js using Rhino.  I switch to running r.js in node, and it worked perfectly.

**Comment by eeroan** [comment]
I did a fork and pull request of version that supports both, but it's still waiting for merge. 

## 

Eero Anttila
+358 50 359 0079

Andy White notifications@github.com kirjoitti 21.11.2012 kello 2.18:

**Comment by dawsontoth** [comment]
Does my fork work with Rhino? I've never used r.js with it. Interested to know. :) https://github.com/dawsontoth/requirejs-tpl/blob/master/tpl.js

**Comment by eeroan** [comment]
Yes it does. I have tested. The only rhino specific behavior is in ajax calls in text.js

-- 
Eero

-------- Alkuperäinen viesti --------
Lähettäjä: Dawson Toth notifications@github.com 
Päivämäärä:  
Saaja: ZeeAgency/requirejs-tpl requirejs-tpl@noreply.github.com 
Kopio: Eero Anttila eea@iki.fi 
Aihe: Re: [requirejs-tpl] Not working when doing bundle with java (#5) 

Does my fork work with Rhino? I've never used r.js with it. Interested to know. :) https://github.com/dawsontoth/requirejs-tpl/blob/master/tpl.js
